src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/types/ReligionCode.kt

```diff
@@ -0,0 +1,115 @@
+package uk.gov.justice.digital.hmpps.personrecord.model.types
+
+enum class ReligionCode(
+  val description: String,
+) {
+  ADV("Adventist"),
+  AGNO("Agnostic"),
+  APO("Apostolic"),
+  ATHE("Atheist"),
+  BAHA("Baháʼí"),
+  BAPT("Christian - Baptist"),
+  BLAC("Black Muslim"),
+  BUDD("Buddhist"),
+  CALV("Calvinist"),
+  CCOG("Celestial Church of God"),
+  CE("Christadelphian"),
+  CHJCLDS("Church of Jesus Christ L D S"),
+  CHRANG("Anglican"),
+  CHRBAPT("Baptist"),
+  CHRCE("Church of England"),
+  CHRCHDN("Christadelphian"),
+  CHRCHSC("Christian Scientist"),
+  CHRCHWA("Church in Wales"),
+  CHRCI("Church of Ireland"),
+  CHRCN("Church of Norway"),
+  CHRCS("Church of Scotland"),
+  CHRCSW("Church of Sweden"),
+  CHREODX("Ethiopian Orthodox"),
+  CHREPIS("Episcopalian"),
+  CHRJEHV("Jehovahs Witness"),
+  CHRLUTH("Lutheran"),
+  CHRMETH("Methodist"),
+  CHRMORM("Church of Jesus Christ of latterday St's"),
+  CHROCR("Other Christian Religion"),
+  CHRORTH("Orthodox (Greek/Russian)"),
+  CHRPENT("Pentecostal"),
+  CHRPRES("Presbyterian"),
+  CHRPROT("Protestant"),
+  CHRRC("Roman Catholic"),
+  CHRSALV("Salvation Army"),
+  CHRSDAY("Seventh Day Adventist"),
+  CHRST("Christian - Other"),
+  CHRODX("Christian - Orthodox"),
+  CHSC("Church of Christ, Scientist"),
+  CINW("Church In Wales"),
+  COFE("Christian - Anglican (incl. CofE & CinW)"),
+  COFI("Church of Ireland"),
+  COFN("Church of Norway"),
+  COFS("Church of Scotland"),
+  CONG("Congregational"),
+  COPT("Coptic"),
+  CSW("Church of Sweden"),
+  DRC("Dutch Reform Church"),
+  DRU("Druid"),
+  EODX("Dutch Reform Church"),
+  EORTH("Eastern Orthodox"),
+  EPIS("Episcopalian"),
+  ETHO("Ethiopian Orthodox"),
+  EVAN("Evangelical"),
+  GOSP("Gospel"),
+  GROX("Greek Orthodox"),
+  HARE("Hare Krishna"),
+  HIND("Hindu"),
+  HNDHAR("Hindu/Hare Krishna"),
+  HUM("Humanist"),
+  JAIN("Jain"),
+  JEHV("Jehovah's Witness"),
+  JEW("Jewish"),
+  LUTH("Lutheran"),
+  METH("Christian - Methodist"),
+  MORM("Mormon"),
+  MUSOTH("Muslim - Other"),
+  MOS("Muslim"),
+  REL01("Not Disclosed"),
+  NIL("No Faith or Belief"),
+  NONC("Non Conformist"),
+  NONP("Non Permitted Religions"),
+  OORTH("Christian - Oriental Orthodox"),
+  ORTH("Orthodox (Greek/Russian)"),
```

Comment:
some of these Christian religions contain the word "Christian", and some do not. The descriptions are very inconsistent.